cli/src/Api.ts

```diff
@@ -509,20 +509,22 @@ export default class Api {
     const distributionBucketIds = []
 
     for (const { id, buckets } of families || []) {
-      const bucketsCountPolicy = distributionBucketFamiliesPolicy.get(id as unknown as DistributionBucketFamilyId)
-      if (bucketsCountPolicy && bucketsCountPolicy.toNumber() < buckets.length) {
-        throw new CLIError(`Distribution buckets policy constraint unsatifified. Not enough distribution buckets exist`)
+      const bucketsCountPolicy = distributionBucketFamiliesPolicy
+        .get(id as unknown as DistributionBucketFamilyId)
+        ?.toNumber()
```

Comment:
Currently it always returns `undefined`
```suggestion
      const bucketsCountPolicy = [...distributionBucketFamiliesPolicy]
        .find(([familyId]) => familyId.toString() === id)?.[1]
        .toNumber()
```